packages/react-ui-core/src/Gmap/Gmap.js

```diff
@@ -124,13 +128,17 @@ export class Gmap extends PureComponent {
     const {
       zoom,
       center,
+      minZoom,
+      maxZoom,
       clickableIcons,
       stylingFunction,
     } = this.props
 
     this.map = new window.google.maps.Map(this.googleMap.current, {
       zoom,
       clickableIcons,
+      minZoom,
+      maxZoom,
```

Comment:
Add unit tests:

* minZoom prop is not set --> it uses the default minZoom
* minZoom prop is set --> it uses the provided minZoom
* maxZoom prop is not set --> it uses the default maxZoom
* maxZoom prop is set --> it uses the provided maxZoom

Reply:
Tests added.